addons/sale/static/src/js/sale_portal_link_payment.js

```diff
@@ -0,0 +1,73 @@
+/** @odoo-module **/
+
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+publicWidget.registry.PortalLinkPayment = publicWidget.Widget.extend({
+    selector: '.o_portal_sale_sidebar',
+    events: Object.assign({}, publicWidget.Widget.prototype.events, {
+        'click button[name="o_sale_portal_amount_installment_button"]': '_onClickAmountInstallmentButton',
+        'click button[name="o_sale_portal_amount_confirmation_button"]': '_onClickAmountConfirmationButton',
+    }),
+
+    start: async function () {
```

Comment:
why not simpler version for all functions?
```suggestion
    async start() {
```